### DISCUSSION TITLE
Please help me fix my wiki

### DISCUSSION THREAD

**Discussion by maxpen** [requarks/wiki#6761] [Help / Questions]
Hello there,

https://docs.requarks.io/install/ubuntu

My installation was working until I tried to fix my SSL that expired to become active again. I tried installing certbot and after that my installation became inaccesible. I'm using docker from the installation instructions. I checked and the image is running and docker is running fine. Been at this for hours and it is getting really frustrating.... I have important data on the wiki so I really need help urgently to get this site up and running again. I've tried restarting and stopping the container. No luck.  I can't even access the site using serverip:3000

Update: I've tried removing the wiki container and re-installing it with the SSL option or without. **Without it works fine.** 

I waited an hour because I was rate limited by letsencrypt and now I get :
**Oops, something went wrong...
Unknown authentication strategy "jwt"**

![image](https://github.com/requarks/wiki/assets/3941651/74082b8a-5e3b-4339-bb50-4a579ffe0b1a)

![image](https://github.com/requarks/wiki/assets/3941651/4c18c3da-9409-49e4-a912-0296f013cfbe)

![image](https://github.com/requarks/wiki/assets/3941651/250a9822-9b58-4f0a-95e0-bd86f9b3ebb6)

**After many hours I tried again installing the ssl version of wiki js and again I'm rate limited:**

![image](https://github.com/requarks/wiki/assets/3941651/25519461-eb88-4a9c-a835-182cfdbe9e24)

**Comment by cdn0x12** [comment]
1. You are subject to rate limiting because your wiki instance is constantly restarting. To remove the rate limit, you need to completely stop the wiki service.
2. The initial error seems to be "The authentication must be pending". At this point, ACME validation has already failed, but the built-in Let's Encrypt client in the wiki is still attempting.
3. Certbot and the built-in Let's Encrypt in the wiki instance can only be enabled one at a time. You may need to explicitly specify the absolute path of the certificate in the wiki configuration if you're using certbot.

**Comment by maxpen** [comment]
1) which command? "docker stop wiki"? I tried doing that many times over
2) okay.....
3) I didn't use certbot the first time when setting it up so maybe I should just remove it and that might solve it? How do I find the certificate path and where is the wiki config files? I mean it runs out of a container on docker so no files...

**Comment by maxpen** [comment]
Please anyone????
